tajo-docs/src/main/sphinx/configuration/storage-site-json.rst

```diff
@@ -0,0 +1,57 @@
+**************************
+The storage-site.json File
+**************************
+
+This file is for configuring :doc:`table spaces</table_management/tablespaces>`.
+The syntax of ``storage-site.json`` is like this basically:
+
+.. code:: json
+
+  {
+    "spaces": {
+      <space config 1>, <space config 2>, ...
+    },
+    "storages": {
+      <storage config 1>, <storage config 2>, ...
+    }
+  }
+
+------
+Spaces
+------
+
+This is a section for registering table spaces. Some space config example is here:
+
+.. code:: json
+
+  "spaces": {
+    "hbase_cluster1": {
+      "uri": "hbase://quorum1:port,quorum2:port/",
+      "config": { ... }
```

Comment:
It would be great if we give some examples to set multiple config key-value pairs.